monai/networks/layers/factories.py

```diff
@@ -82,15 +82,7 @@ class LayerFactory:
     def __init__(self) -> None:
         self.factories: dict[str, Callable] = {}
 
-    @property
-    def names(self) -> tuple[str, ...]:
-        """
-        Produces all factory names.
-        """
-
-        return tuple(self.factories)
-
-    def add_factory_callable(self, name: str, func: Callable) -> None:
```

Comment:
I would keep this name if even just as an alias so that it's clearer what we're adding to the factory.